pkg/appregistry/appregistry_test.go

```diff
@@ -0,0 +1,22 @@
+package appregistry_test
```

Comment:
Why a new package for testing?

Reply:
I am treating this test as a black box test, hence a new package for test(s) and explicit import of package being tested.